runtime/src/test_utils.rs

```diff
@@ -1166,25 +1100,6 @@ impl<BS: Blockstore> Runtime for MockRuntime<BS> {
         &self.store
     }
 
-    fn send(
```

Comment:
We now rely on the default implementation, which delegates to `send_generalized`.